src/app/features/location/address-search/address-search.component.scss

```
@@ -7,6 +7,10 @@
     ::ng-deep .mat-mdc-form-field-infix {
       position: static;
     }
+
+    ::ng-deep .mat-mdc-autocomplete-panel {
```

Stylelint (17.13.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)